README.md

```diff
@@ -22,6 +22,22 @@ Chart for s3manager - web GUI for s3
 helm repo add sergeyshevch https://sergeyshevch.github.io/charts
 helm upgrate --install sergeyshevch/s3manager-helm RELEASE_NAME
 ```
+## deploy
```

Comment:
Please edit README.md.gotmpl instead and run helm-docs from Makefile. Also fill changelog please